tests/test.py

```diff
@@ -88,19 +89,12 @@ def test_predict():
 def test_predict_other_formats():
     '''Check the prediction output of JPEG and TIFF converted images.'''
 
-    # save img as jpeg
-    Image.open('samples/test_examples/low_resolution/woman.png').save(
-        'samples/test_examples/low_resolution/woman.jpeg', 'JPEG')
-
-    # save img as tiff
-    Image.open('samples/test_examples/low_resolution/woman.png').save(
-        'samples/test_examples/low_resolution/woman.tiff', 'TIFF')
-
-    im = call_model(file_path='samples/test_examples/low_resolution/woman.jpeg')
-    assert im.size == (424, 636)
-
-    im = call_model(file_path='samples/test_examples/low_resolution/woman.tiff')
-    assert im.size == (424, 636)
+    with Image.open('samples/test_examples/low_resolution/woman.png') as img:
+        for ext in ('jpeg', 'tiff'):
+            with NamedTemporaryFile(suffix='.' + ext) as tmp_img:
+                img.save(tmp_img)
```

Comment:
I forgot to add the format: `img.save(tmp_img, ext)`. Using this occasion, we can perhaps also add `gif`.